repositories.md

```diff
@@ -235,3 +235,14 @@ In setting up an Uptane system, an implementer will need to specify how informat
 To facilitate coordination between implementations, an Uptane adopter can choose to write a POUF, an added layer to the Standard in which an implementer can specify choices of Protocols, Operations, Usage and Formats. A POUF provides an easy way for an implementer to specify the elements that can ensure interoperability. It can also be customized for the special needs of fleet owners in a particular industry, such as taxis, car sharing networks, police forces, or the military.
 
 Information on writing a POUF can be found on the [POUF Purpose and Guidelines](https://uptane.github.io/pouf.html) page of the Uptane website. A sample POUF, written for the [Uptane Reference Implementation](https://uptane.github.io/reference_pouf.html), offers sample metadata written in [ASN.1/DER](https://github.com/uptane/uptane.github.io/blob/master/reference_pouf.md#file-formats).
+
+## Cost Considerations
+
+There is no short answer to the question of what it costs to implement Uptane because such a decision has a number of variables to be considered, starting with how they plan to do this implementation.  Currently, an  OEM has two methods to implement Uptane: either buy an off-the-shelf solution and do an integration, or build a custom solution, with some greater or smaller degree of reliance on the open-source client and server components. If a project is starting from scratch, there is a third option. Uptane could be integrated in the design stage. At that point, the cost would be minimal and any money spent would be just one component of the initial design. But, most OEMS contemplating an integration will not be at that initial stage. 
+
+If a user is contemplating options one or two, the relative costs become admittedly more difficult to estimate. In the case of off-the-shelf products, the goals of the provider and the OEM could be somewhat misaligned, making the integration difficult and likely expensive. Additionally, there is always a risk profile associated with getting an OTA system from a 3rd party, as the supplier could go out of business, or decide to phase out the product in the wake of a shift in corporate priority.
+
+Arguably then, some variant of the custom in-house option is the safest choice for larger OEMs. But it will mean higher cost and greater effort. 
```

Comment:
```suggestion
If a user is contemplating options one or two, the relative costs are admittedly more difficult to estimate. When considering off-the-shelf products, an Uptane solution may be more or less expensive than any other OTA solution. Uptane itself may not be the primary driver of cost in this case. There is also the possibility that the goals of the provider and the OEM could be misaligned, which can make integration difficult and likely expensive. Additionally, there is always a risk profile associated with outsourcing an OTA system to a third party, as the supplier could go out of business or decide to phase out the product in the wake of a shift in corporate priority.

A variant of the custom in-house option may be a safer choice for larger OEMs, but it may also come with a higher cost and greater effort. However, when upgrading from a legacy solution, a risk assessment of the current solution and evaluation of the current weaknesses and opportunities for improvement will be necessary in any case. Uptane provides an organized framework for this assessment, and thus may even reduce costs compared to an assessment with such a framework. The costs of the upgrade itself are dependent upon the individual needs, security concerns, and existing infrastructure of the implementer.
```